script/release/release-containerd.sh

```
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+#
+# Releases and cross compile containerd.
+#
+set -eu -o pipefail
+
+install_dependencies() {
+  dpkg --add-architecture ${1}
+  apt-get install crossbuild-essential-${1}
+  apt-get install libseccomp-dev:${1}
+}
+
+# Add repositories with multiple architectures
+source /etc/os-release
+cat <<EOF > /etc/apt/sources.list
```

I think the script should fail when `$CI` is not set.

---

Or wondering we can use Dockerfile here